src/test/java/org/mule/extension/ftp/FtpDirectoryListenerFunctionalTestCase.java

```diff
@@ -127,7 +127,7 @@ public void matcher() throws Exception {
     testHarness.write(rejectedFile.getPath(), WATCH_CONTENT);
 
     assertPoll(file, DR_MANHATTAN);
-    checkNot(PROBER_TIMEOUT, PROBER_DELAY, () -> RECEIVED_MESSAGES.size() > 1);
+    checkNot(50000, PROBER_DELAY, () -> RECEIVED_MESSAGES.size() > 1);
```

Comment:
Why does increasing the timeout fix this test? The check there is to ensure that something does not happen, if it does not happen in 50 seconds it will not happen during PROBER_TIMEOUT which is 10 seconds.